### PR TITLE
add check if participant id is altpid to get guid from it

### PR DESCRIPTION
**Pull request by gmakharat**

### CODE REVIEW

File: src/main/java/org/broadinstitute/dsm/route/PatchRoute.java [hunk 1]
```diff
@@ -22,6 +22,7 @@
 import org.broadinstitute.dsm.model.NameValue;
 import org.broadinstitute.dsm.model.Patch;
 import org.broadinstitute.dsm.model.Value;
+import org.broadinstitute.dsm.model.elasticsearch.ESProfile;
```
Comment (SimoneMaiwald) [inline src/main/java/org/broadinstitute/dsm/route/PatchRoute.java]: I don't think this is used

File: src/main/java/org/broadinstitute/dsm/util/ElasticSearchUtil.java [hunk 2]
```diff
@@ -44,6 +44,7 @@
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
+import java.awt.color.ICC_ColorSpace;
```
Comment (SimoneMaiwald) [inline src/main/java/org/broadinstitute/dsm/util/ElasticSearchUtil.java]: ?

File: src/main/java/org/broadinstitute/dsm/route/PatchRoute.java (hunk 1 above)
Comment (gmakharat) [reply src/main/java/org/broadinstitute/dsm/route/PatchRoute.java]: actually it is used on line 612

Comment (SimoneMaiwald) [reply src/main/java/org/broadinstitute/dsm/route/PatchRoute.java]: File I am looking at only has 527 lines

**Comment by yufengwng** [comment]
I think this PR is old. Looks like the code is already in `develop` now.

**Pull request closed without merging**